src/base-element.js

```diff
@@ -862,6 +862,24 @@ export class BaseElement {
   }
 
   /**
+   * Expands the element, resetting its default display value, and notifies its
+   * owner (if there is one) through {@link expandedCallback} that the element
+   * is no longer visible.
+   */
+  expand() {
+    this.element.getResources().expandElement(this.element);
+  }
+
+  /**
+   * Called every time an owned AmpElement expands itself.
+   * See {@link expand}.
+   * @param {!AmpElement} unusedElement
```

Comment:
What is this argument actually? The element itself? If so, we already have `this.element` so it's not needed.